XRTK-Core/Packages/com.xrtk.core/Runtime/EventDatum/Diagnostics/BaseDiagnosticsEventData.cs

```diff
@@ -5,7 +5,7 @@
 using UnityEngine.EventSystems;
 using XRTK.Extensions;
 
-namespace XRTK.EventDatum.DiagnosticsSystem
+namespace XRTK.EventDatum.Diagnostics
```

Comment:
Not sure if we should include this kind of breaking change in with the Locomotion changes.